subsys/net/ip/net_context.c

```diff
@@ -52,6 +52,7 @@ static struct net_context contexts[NET_MAX_CONTEXT];
  */
```

Comment:
The "net: Fix compilation warnings" is in a way correct subject, but I think it does not reflect the spirit of this patch. Better subject would be "net: Allow disabling IPv4, IPv6, UDP and TCP simultaneously". The body then tells that we fix the complation warnings.

Reply:
I thought it wouldn't fit long text like that. Tried to shrink it and the result was not clear. Thanks, updated it.